docs/learn/injecting-css.mdx

```diff
@@ -5,23 +5,23 @@ sidebar_position: 3
 
 # Injecting CSS
 
-Argos allows custom CSS injection in screenshots for more control over visual output.
+Learn how to inject CSS before taking a screenshot.
 
-In addition to [Argos built-in helpers](https://argos-ci.com/docs/argos-helpers#helpers), Argos integrations provide the capability to inject custom CSS into your screenshots. This feature is particularly useful in complex scenarios for excluding certain elements or addressing flaky visual differences.
+In addition to [Argos built-in helpers](https://argos-ci.com/docs/argos-helpers#helpers), Argos offer the possibility to inject custom CSS applied during the screenshot process. It can be helpful for complex use cases that produce flaky visual diffs.
```

Comment:
```suggestion
In addition to [built-in helpers](https://argos-ci.com/docs/argos-helpers#helpers), Argos offers the possibility to add custom CSS evaluated during the screenshot process. It can be helpful for complex use cases that produce flaky visual diffs.
```